### DISCUSSION TITLE
Trilium Web Clipper Not accessed in Microsoft Edge Addons?

### DISCUSSION THREAD

**Discussion by zhang-stephen** [TriliumNext/Trilium#3224] [Q&A] [answered]
I cannot find this plugins in [Microsoft Edge Addons](https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home).

**Comment by zadam** [comment, marked as answer]
The Web Clipper is published only to Firefox and Chrome addon registries.

You can download the chrome version from the https://github.com/zadam/trilium-web-clipper/releases - I believe it should be compatible with Edge.

**Reply by zhang-stephen** [reply]
Hi, thank you for the reply.
Yes, I tried and it is compatible with Microsoft Edge. But what's the username with docker deployment on cloud server? I found nothing in the wiki... 😕

**Reply by zadam** [reply]
Use e.g. "trilium". In 0.50 username was removed and is ignored.

https://github.com/zadam/trilium/wiki/Web-clipper#username

**Reply by zhang-stephen** [reply]
Thanks~
